src/m1_facilitator/Directory.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import path from 'path';
+import os from 'os';
+
+/**
+ * Provide method to calculate folder and file path.
+ */
+export default class Directory {
```

Comment:
The class name is confusing. Why it's named `Directory`?

Reply:
It's named Directory, because it deal with folder and file path.  This naming was previously agreed. We can definitely improve, if there is any suggestion.

Reply:
Let us keep it as `Directory` for now.